ep/src/lib.rs

```diff
@@ -47,21 +50,20 @@ pub struct Epaddr {
   mctxt_nsconn: usize,
   mctxt_send_seqnum: u16,
   mctxt_recv_seqnum: u16,
-  mctxt_current: Option<Rc<Epaddr>>,
+  mctxt_current: Option<&'a Epaddr<'a>>,
```

Comment:
I just noticed that this particular field is sometimes used as a reference to a single epaddr and then other times pointing to the first item in an array on the stack..... gj C. Going to have to investigate more if they treat it that way. Ill put a comment about this.